internal/cliwrapper/cliwrapper_test.go

```diff
@@ -17,20 +17,20 @@ type TestModule struct {
 }
 
 func GetPythonPath() (string, error) {
-	python3Path, errPython3 := exec.LookPath("python3")
-	if errPython3 != nil {
-		pythonPath, errPython := exec.LookPath("python")
-		if errPython != nil {
-			return "", fmt.Errorf("error getting Python3 (%s) and python (%s)", errPython3, errPython)
-		}
-		return pythonPath, nil
+	var errP3, errP error
+	if p, errP3 := exec.LookPath("python3"); errP3 == nil {
+		return p, nil
 	}
-	return python3Path, nil
+	if p, errP := exec.LookPath("python"); errP == nil {
+		return p, nil
+	}
+	return "", fmt.Errorf("errors getting paths for Python3 (%s) and python (%s)",
+		errP3.Error(), errP.Error())
```

Comment:
Would it be better to use the appropriate formatting "verb" (e.g., `%w` or `%v` instead of `%s`) rather than using explicit calls to `Error()`?

Ditto in `internal/connector/connector_test.go`.

Reply:
I think I wrote that line (since it's in the unchanged code). That could work. Though it will have some extra characters cluttering it up. Either way works.

Reply:
oh it would definitely be better to wrap the errors and use %w, but that's also not for this pr

Reply:
@jaredoconnell, the uses of `Error()` here are new (@mfleader added them after wondering why the linter did not object to having two `error`'s formatted in the same message).

@mfleader, I bow to your choice.